app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/item/VaultItemIdentityContent.kt

```diff
@@ -284,3 +247,35 @@ fun VaultItemIdentityContent(
         }
     }
 }
+
+@Composable
+private fun IdentityCopyField(
+    label: String,
+    value: String,
+    contentDescription: String,
+    textFieldTestTag: String,
+    copyActionTestTag: String,
+    onCopyClick: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Spacer(modifier = Modifier.height(8.dp))
+    BitwardenTextFieldWithActions(
+        label = label,
+        value = value,
+        onValueChange = { },
+        readOnly = true,
+        singleLine = false,
+        actions = {
+            BitwardenTonalIconButton(
+                vectorIconRes = R.drawable.ic_copy,
+                contentDescription = contentDescription,
+                onClick = onCopyClick,
+                modifier = Modifier.testTag(tag = copyActionTestTag),
+            )
+        },
+        modifier = modifier
+            .fillMaxWidth()
+            .padding(horizontal = 16.dp),
```

Comment:
Things like this should be passed in from the top, the inner component should not know that it needs this padding or fillMaxWidth.